### PR TITLE
Final Capstone Group Project Front-End: Car Booking App

### PR DESCRIPTION
**Pull request by cynthiainga**
**The learning objectives of this project are:**

- [x] Apply technical knowledge and skills gained in previous modules in a complex project.
- [x] Understand pros and cons of different approaches of connecting Ruby on Rails back-end with React front-end.
- [x] Understand principles of Ruby on Rails and React frameworks.
- [x] Apply JavaScript best practices and language style guides in code.
- [x] Apply React best practices and language style guides in code.
- [x] Learn about and practice giving constructive feedback to teammates.
- [x] Perform a code review for a team member.
- [x] Use the "Review change"" feature from GitHub.
- [x] Write clear, professional, and respectful review comments for other team members.
- [x] Explain "why" a change is requested when giving a code review.
- [x] Plan a 2+ week project with no interim Microverse-set milestones and submit it on time.
- [x] Apply knowledge of setting working agreements to set group project teams up for success.
- [x] Independently implement best practices for group agreements to improve teamwork in larger group project.
- [x] Demonstrate ability to apply best practices of communication for resolving teamwork challenges.
- [x] Understand that respect is the foundation of strong relationship-building with teammates.
- [x] Show up throughout group projects as a reliable and committed team member who communicates and manages your time effectively.
- [x] Recognize the importance of investing in good working relationships with teammates.
- [x] Understand principles of strong teamwork (being reliable, committed, and consistent) and how to apply them in group projects.
- [x] Recognize the value of making equal contributions to group projects to produce the best outcome.
- [x] Use empty Kanban board to manage tasks with the team and own time on the project.

## [Link to the Back-End repo](https://github.com/Tobinchilongo/Final-Capstone-BackEnd)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,152 @@
...
+  <h1 align="center">Rent A Car </h1>
 
-In the project directory, you can run:
+  <p align="center">
+    A car rental website
+    <br />
 
-### `npm start`
+## Back-end Repository Link
+<a href="https://github.com/Tobinchilongo/Final-Capstone-BackEnd.git">Book A Car Frontend</a>
```
Comment (mcrd25) [inline README.md:17]: Should definitely change from Book A Car to Frontend to what Book a Car Back-end

File: README.md [hunk 2]
```diff
@@ -1,70 +1,152 @@
...
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+## API link
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+[Cars](https://agile-peak-14047.herokuapp.com/api/v1/cars)
+[Reservations](https://agile-peak-14047.herokuapp.com/api/v1/reservation)
+## Live Demo
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+[swagger-ui](https://agile-peak-14047.herokuapp.com/api-docs/index.html)
```
Comment (mcrd25) [inline README.md:42]: Live demo should probably be Front-End related

File: README.md [hunk 3]
```diff
@@ -1,70 +1,152 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+Setup the database with:
+```bash
+rails db:create
+```
+<div>OR</div>
 
-### Making a Progressive Web App
+```bash
+rake db:create
```
Comment (mcrd25) [inline README.md:72]: Remember focus is front-end

**Review by mcrd25 — CHANGES_REQUESTED** [review]
Hi @cynthiainga @Tobinchilongo @jerryowusu , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there! The main thing is the README, can definitely tell you took the Back-end README and edited it, which is fine, but some of the info is a bit mismatch.



## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Make the site look as close to the original as possible, remember, the point is for you to follow the design. In the real world, you follow the design the client has laid out for you as closely as possible unless your job is to do the design yourself.
- writing tests


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 2 above)
Comment (jerryowusu) [reply README.md:42]: Thanks for notifying us. It was definitely an oversight. Resolved

**Comment by jerryowusu** [comment]
Thanks a lot @mcrd25 for the candid reviews. we really appreciate all of them.

**Pull request merged**